dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/AlertGroupControllerTest.java

```diff
@@ -210,7 +210,7 @@ public void test090DelAlertGroupById() throws Exception {
         MvcResult mvcResult = mockMvc.perform(delete("/alert-groups/1")
                 .header("sessionId", sessionId)
                 .params(paramsMap))
-                .andExpect(status().isOk())
+                .andExpect(status().isInternalServerError())
```

Comment:
This test case seems not good, why `test080DelAlertGroupById` is successful but `test090DelAlertGroupById` throws a exception?